common/config.py

```diff
@@ -1626,7 +1626,7 @@ def createNewCrontab(self, oldCrontab):
             # entries if there is no automatic entry.
             newCrontab.append(self.SYSTEM_ENTRY_MESSAGE)
             newCrontab.append("#Please don't delete these two lines, or all custom backintime "
-                              "entries are going to be deleted next time you call the gui options!")
+                              "entries will be deleted next time you call the gui options!")
```

Comment:
Wow, this is a high quality review! Thank you!

BiT uses this phrase to test for own entries in `crontab`. We need to check if changing this phrase will break that test!

Reply:
I added a docstring incl. WARNING to SYSTEM_ENTRY_MESSAGE.